{
    "type": "bundle",
    "id": "bundle--a6c2acbe-cab1-4715-ac54-1d25492e04e4",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--8fd02ad9-d3a9-45a9-b39a-f579a6468730",
            "created_by_ref": "identity--748e6444-f073-4c50-b558-f49be8897a81",
            "created": "2022-11-09T20:15:00.000Z",
            "modified": "2022-11-10T14:28:00.000Z",
            "name": "CVE-2022-42965",
            "description": "An exponential ReDoS (Regular Expression Denial of Service) can be triggered in the snowflake-connector-python PyPI package, when an attacker is able to supply arbitrary input to the get_file_transfer_type method",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-42965"
                },
                {
                    "source_name": "cwe",
                    "external_id": "NVD-CWE-Other"
                },
                {
                    "source_name": "https://research.jfrog.com/vulnerabilities/snowflake-connector-python-redos-xray-257185/",
                    "url": "https://research.jfrog.com/vulnerabilities/snowflake-connector-python-redos-xray-257185/"
                },
                {
                    "source_name": "cve2stix",
                    "description": "This object was created using cve2stix from the Signals Corps.",
                    "url": "https://github.com/signalscorps/cve2stix"
                }
            ],
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ],
            "extensions": {
                "extension-definition--b2b5f2cd-49e6-4091-a0e0-c0bb71543e23": {
                    "extension_type": "property-extension",
                    "nvd_cve": {
                        "data_type": "CVE",
                        "data_format": "MITRE",
                        "data_version": "4.0",
                        "CVE_data_meta": {
                            "ID": "CVE-2022-42965",
                            "ASSIGNER": "security@jfrog.com"
                        },
                        "problemtype": {
                            "problemtype_data": [
                                {
                                    "description": [
                                        {
                                            "lang": "en",
                                            "value": "NVD-CWE-Other"
                                        }
                                    ]
                                }
                            ]
                        },
                        "references": {
                            "reference_data": [
                                {
                                    "url": "https://research.jfrog.com/vulnerabilities/snowflake-connector-python-redos-xray-257185/",
                                    "name": "https://research.jfrog.com/vulnerabilities/snowflake-connector-python-redos-xray-257185/",
                                    "refsource": "MISC",
                                    "tags": [
                                        "Exploit",
                                        "Third Party Advisory"
                                    ]
                                }
                            ]
                        },
                        "description": {
                            "description_data": [
                                {
                                    "lang": "en",
                                    "value": "An exponential ReDoS (Regular Expression Denial of Service) can be triggered in the snowflake-connector-python PyPI package, when an attacker is able to supply arbitrary input to the get_file_transfer_type method"
                                }
                            ]
                        }
                    },
                    "impact": {
                        "baseMetricV3": {
                            "cvssV3": {
                                "version": "3.1",
                                "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
                                "attackVector": "NETWORK",
                                "attackComplexity": "LOW",
                                "privilegesRequired": "NONE",
                                "userInteraction": "NONE",
                                "scope": "UNCHANGED",
                                "confidentialityImpact": "NONE",
                                "integrityImpact": "NONE",
                                "availabilityImpact": "HIGH",
                                "baseScore": 7.5,
                                "baseSeverity": "HIGH"
                            },
                            "exploitabilityScore": 3.9,
                            "impactScore": 3.6
                        }
                    },
                    "publishedDate": "2022-11-09T20:15Z",
                    "lastModifiedDate": "2022-11-10T14:28Z"
                }
            }
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--c855f4d0-6af4-48d2-8bea-f9d0b682cd1c",
            "created_by_ref": "identity--748e6444-f073-4c50-b558-f49be8897a81",
            "created": "2022-11-09T20:15:00.000Z",
            "modified": "2022-11-10T14:28:00.000Z",
            "name": "CVE-2022-42965",
            "description": "An exponential ReDoS (Regular Expression Denial of Service) can be triggered in the snowflake-connector-python PyPI package, when an attacker is able to supply arbitrary input to the get_file_transfer_type method",
            "indicator_types": [
                "compromised"
            ],
            "pattern": "[(software:cpe='cpe:2.3:a:snowflake:snowflake-connector-python:-:*:*:*:*:*:*:*')]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-11-09T20:15:00Z",
            "external_references": [
                {
                    "source_name": "cve2stix",
                    "description": "This object was created using cve2stix from the Signals Corps.",
                    "url": "https://github.com/signalscorps/cve2stix"
                }
            ],
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ],
            "extensions": {
                "extension-definition--b463c449-d022-48b7-b464-3e9c7ec5cf16": {
                    "extension_type": "property-extension",
                    "all_cpe23uris": [
                        "cpe:2.3:a:snowflake:snowflake-connector-python:-:*:*:*:*:*:*:*"
                    ],
                    "vulnerable_cpeid": [
                        "cpe:2.3:a:snowflake:snowflake-connector-python:-:*:*:*:*:*:*:*"
                    ]
                }
            }
        },
        {
            "type": "relationship",
            "spec_version": "2.1",
            "id": "relationship--18c23eb0-7d9f-43e3-9ba7-7c6bfc71398f",
            "created_by_ref": "identity--748e6444-f073-4c50-b558-f49be8897a81",
            "created": "2022-11-09T20:15:00.000Z",
            "modified": "2022-11-10T14:28:00.000Z",
            "relationship_type": "identifies",
            "source_ref": "indicator--c855f4d0-6af4-48d2-8bea-f9d0b682cd1c",
            "target_ref": "vulnerability--00538c40-3f07-4ba8-8825-12ae828f04b7",
            "external_references": [
                {
                    "source_name": "cve2stix",
                    "description": "This object was created using cve2stix from the Signals Corps.",
                    "url": "https://github.com/signalscorps/cve2stix"
                }
            ]
        }
    ]
}